### PR TITLE
Three new links are added in open source projects file

### LINKED ISSUE #699: [Add] Add links to websites which have a great list of open-source projects
### Site URL

https://www.gsocorganizations.dev/

### Description of the above provided site

A site for viewing and analyzing the info of the organizations participating in Google Summer of Code.

👉 A couple of similar websites are also there which I want to add.

### Category

Open Source

### Sub Category

Projects

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by Isha988**
## Fixes Issue
Closes #699 

## Changes proposed
Three websites are added
1. **GSoC Organizations:** https://www.gsocorganizations.dev/
2. **Up for grabs:** https://up-for-grabs.net/
3. **CodeTriage:** https://www.codetriage.com/

### CODE REVIEW

File: database/opensource/projects.json [hunk 1]
```diff
@@ -47,5 +47,26 @@
...
+  {
+    "name": "GSoC Organizations",
+    "description": "A site for viewing and analysing the info of the organizations participating in Google Summer of Code and filtering them by various factors like years of participation, categories, etc.",
+    "url": "https://www.gsocorganizations.dev/",
+    "category": "opensource",
+    "subcategory": "projects"
+  },
+  {
+    "name": "Up for grabs",
+    "description": "This is a list of projects which have curated tasks specifically for new contributors. These are a great way to get started with a project, or to help share the load of working on open source projects. ",
```
Comment (CBID2) [inline database/opensource/projects.json:60]: [Typo]
```suggestion
    "description": "This is a list of projects which have curated tasks specifically for new contributors. These are a great way to get started with a project, or help share the load of working on open source projects. ",
```

File: database/opensource/projects.json [hunk 2]
```diff
@@ -47,5 +47,26 @@
...
+    "url": "https://up-for-grabs.net/",
+    "category": "opensource",
+    "subcategory": "projects"
+  },
+  {
+    "name": "CodeTriage",
+    "description": "Pick your favorite repos to receive a different open issue in your inbox every day. Fix the issue and everybody wins. 71,669 developers are working on 7,712 open source repos using CodeTriage.",
```
Comment (CBID2) [inline database/opensource/projects.json:67]: [Proofreading]
```suggestion
    "description": "Pick your favorite repos and receive a different open issue in your inbox every day. Fix the issue and everybody wins! 71,669 developers are working on 7,712 open source repos using CodeTriage.",
```

File: database/opensource/projects.json [hunk 3]
```diff
@@ -47,5 +47,26 @@
     "url": "https://www.codecademy.com/pages/contribute-docs",
     "category": "opensource",
     "subcategory": "projects"
+  },
+  {
+    "name": "GSoC Organizations",
+    "description": "A site for viewing and analysing the info of the organizations participating in Google Summer of Code and filtering them by various factors like years of participation, categories, etc.",
```
Comment (CBID2) [inline database/opensource/projects.json:53]: [Proofreading]
```suggestion
    "description": "A site that views and analyzed information on the organizations participating in Google Summer of Code, filtering them by various factors like years of participation, categories, etc.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Isha988! :) I pointed out a few proofreading errors and typos in my inline comments. In the future, consider running your descriptions through Grammarly or Spellcheck before making a pull request. Other than that, you're off to a great start! :)

**Comment by CBID2** [comment]
Hi @Isha988! 😊 The linked issue refers to something that has already been solved, so I’m going to close your pull request for now. Refer to the [Contributing file](https://github.com/rupali-codes/linkshub/blob/main/CONTRIBUTING.md#issues) to help you create an issue that correlates to this PR. Once that’s done, tag me and it will be opened! 😊

**Comment by Isha988** [comment]
@CBID2 I had linked the wrong issue by mistake, but now that I have tagged the right issue, would you please reopen this PR so that I can make suggested changes?

File: database/opensource/projects.json [hunk 4]
```diff
@@ -50,7 +50,7 @@
   },
   {
     "name": "GSoC Organizations",
-    "description": "A site for viewing and analysing the info of the organizations participating in Google Summer of Code and filtering them by various factors like years of participation, categories, etc.",
+    "description": "A site that views and analyzed information on the organizations participating in Google Summer of Code, filtering them by various factors like years of participation, categories, etc.",
```
Comment (CBID2) [inline database/opensource/projects.json:53]:
```suggestion
    "description": "A site that views and analyzes information on the organizations participating in Google Summer of Code, filtering them by various factors like years of participation, categories, etc.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
So close @Isha988.

**Comment by Isha988** [comment]
> So close @Isha988.

I copied the description from their respective websites, but still, it has such errors.

**Review by CBID2 — APPROVED** [review]
Great job @Isha988! 😄 LGTM! 😄

**Review by rupali-codes — CHANGES_REQUESTED** [review]
Hey Isha, we have just added a new category named _tools_, could you please add _up for grabs_ & _codeTriage_ in there? thanks

File: database/opensource/projects.json [hunk 5]
```diff
@@ -47,5 +47,26 @@
...
+    "url": "https://www.gsocorganizations.dev/",
+    "category": "opensource",
+    "subcategory": "projects"
+  },
+  {
+    "name": "Up for grabs",
+    "description": "This is a list of projects which have curated tasks specifically for new contributors. These are a great way to get started with a project, or help share the load of working on open source projects. ",
+    "url": "https://up-for-grabs.net/",
+    "category": "opensource",
+    "subcategory": "projects"
```
Comment (CBID2) [inline database/opensource/projects.json:63]: [Proofreading] 
```suggestion
    "subcategory": "tools"
```

Comment (CBID2) [inline database/opensource/projects.json:63]:
```suggestion
    "subcategory": "tools"
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Isha988! @rupali-codes requested some changes so add some inline comments to help you.

**Comment by Isha988** [comment]
> Hey Isha, we have just added a new category named _tools_, could you please add _up for grabs_ & _codeTriage_ in there? thanks

Ok, will do

**Review by CBID2 — APPROVED** [review]
Great job @Isha988! 😄 LGTM! 😄

**Comment by Isha988** [comment]
> Great job @Isha988! 😄 LGTM! 😄

Finally 😁😅

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Isha988! :)
Ignore my previous comment. There’s conflicts in your branch that need to be solved.

**Comment by Isha988** [comment]
> Hi @Isha988! :) Ignore my previous comment. There’s conflicts in your branch that need to be solved.

How to resolve them?

**Comment by CBID2** [comment]
> > Hi @Isha988! :) Ignore my previous comment. There’s conflicts in your branch that need to be solved.
> 
> How to resolve them?

I'm not too sure 🤔 @Isha988.

**Comment by Isha988** [comment]
> Hi @Isha988! :) Ignore my previous comment. There’s conflicts in your branch that need to be solved.

I have resolved the conflicts of database/index.ts and database/data.ts files but in remaining two file it is showing the changes that I have made as conflicts.

**Comment by rupali-codes** [comment]
@Isha988 i can not see the option to resolve conflicts could you please enable that so i can help you with that

**Comment by Isha988** [comment]
> @Isha988 I can not see the option to resolve conflicts could you please enable that so I can help you with that

@rupali-codes  is it enabled now?

**Comment by Isha988** [comment]
> Hi @Isha988! :) Ignore my previous comment. There’s conflicts in your branch that need to be solved.

Hey, @CBID2 and @rupali-codes Now that I have resolved all the conflicts, and I think we are good to go with this.

**Review by CBID2 — APPROVED** [review]
Great job @Isha988! 😄 LGTM! 😄

**Review by rupali-codes — APPROVED** [review]
looks perfect, thanks Isha

**Pull request merged**